src/thunderbots/software/ai/hl/stp/stp.cpp

```diff
@@ -0,0 +1,102 @@
+#include "ai/hl/stp/stp.h"
+
+#include <ai/hl/stp/play/play_factory.h>
+#include <munkres/munkres.h>
+
+#include <chrono>
+#include <exception>
+#include <random>
+
+#include "ai/hl/stp/play/play.h"
+#include "ai/hl/stp/tactic/tactic.h"
+#include "ai/intent/move_intent.h"
+#include "util/logger/init.h"
+
+STP::STP(long random_seed)
+    : random_number_generator(random_seed), uniform_distribution(1, 6)
+{
+}
+
+std::vector<std::unique_ptr<Intent>> STP::getIntentAssignment(const World& world)
```

Comment:
perhaps `getIntents` might be a bit better? Took me a second to figure out why this wasn't returning a map.

Reply:
renamed